### ISSUE TITLE
Console Error with Contao 4.8.3

### ISSUE THREAD

**Issue by m-vo** [contao-community-alliance/dependency-container#16]
I get the following error with Contao 4.8.3 when I'm trying to dump detailed config via the console (e.g. with `vendor/bin/contao-console config:dump-reference twig`):

```
Impossible to call set() on a frozen ParameterBag.
In FrozenParameterBag.php line 58:
                                                                    
  [Symfony\Component\DependencyInjection\Exception\LogicException]  
  Impossible to call set() on a frozen ParameterBag.                                                                                    

Exception trace:
  at [...]/www/vendor/symfony/dependency-injection/ParameterBag/FrozenParameterBag.php:58
 Symfony\Component\DependencyInjection\ParameterBag\FrozenParameterBag->set() at [...]/www/vendor/symfony/dependency-injection/Container.php:135
 Symfony\Component\DependencyInjection\Container->setParameter() at [...]/www/vendor/contao-community-alliance/dependency-container/src/CcaDependencyInjectionBundle.php:40
 DependencyInjection\Container\CcaDependencyInjectionBundle->build() at [...]/www/vendor/symfony/framework-bundle/Command/AbstractConfigCommand.php:136
 Symfony\Bundle\FrameworkBundle\Command\AbstractConfigCommand->initializeBundles() at [...]/www/vendor/symfony/framework-bundle/Command/AbstractConfigCommand.php:64
 Symfony\Bundle\FrameworkBundle\Command\AbstractConfigCommand->findExtension() at [...]/www/vendor/symfony/framework-bundle/Command/ConfigDumpReferenceCommand.php:92
 Symfony\Bundle\FrameworkBundle\Command\ConfigDumpReferenceCommand->execute() at [...]/www/vendor/symfony/console/Command/Command.php:255
 Symfony\Component\Console\Command\Command->run() at [...]/www/vendor/symfony/console/Application.php:933
 Symfony\Component\Console\Application->doRunCommand() at [...]/www/vendor/symfony/framework-bundle/Console/Application.php:87
 Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand() at [...]/www/vendor/symfony/console/Application.php:272
 Symfony\Component\Console\Application->doRun() at [...]/www/vendor/symfony/framework-bundle/Console/Application.php:73
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at [...]/www/vendor/symfony/console/Application.php:148
 Symfony\Component\Console\Application->run() at [...]/www/vendor/contao/manager-bundle/bin/contao-console:38
```

**Comment by bezin** [comment]
Hi there, 

any news on this? I ran into the same issue as well: 

```
12:42:09 ERROR     [console] Error thrown while running command "debug:config NelmioCorsBundle -vvv". Message: "Impossible to call set() on a frozen ParameterBag."
[
  "exception" => Symfony\Component\DependencyInjection\Exception\LogicException {
    #message: "Impossible to call set() on a frozen ParameterBag."
    #code: 0
    #file: "C:\dev\web\project\vendor\symfony\symfony\src\Symfony\Component\DependencyInjection\ParameterBag\FrozenParameterBag.php"
    #line: 58
    trace: {
      C:\dev\web\project\vendor\symfony\symfony\src\Symfony\Component\DependencyInjection\ParameterBag\FrozenParameterBag.php:58 { …}
      C:\dev\web\project\vendor\symfony\symfony\src\Symfony\Component\DependencyInjection\Container.php:156 { …}
      C:\dev\web\project\vendor\contao-community-alliance\dependency-container\src\CcaDependencyInjectionBundle.php:40 { …}
      C:\dev\web\project\vendor\symfony\symfony\src\Symfony\Bundle\FrameworkBundle\Command\AbstractConfigCommand.php:136 { …}
      C:\dev\web\project\vendor\symfony\symfony\src\Symfony\Bundle\FrameworkBundle\Command\AbstractConfigCommand.php:64 { …}
      C:\dev\web\project\vendor\symfony\symfony\src\Symfony\Bundle\FrameworkBundle\Command\ConfigDebugCommand.php:78 { …}
      C:\dev\web\project\vendor\symfony\symfony\src\Symfony\Component\Console\Command\Command.php:255 { …}
      C:\dev\web\project\vendor\symfony\symfony\src\Symfony\Component\Console\Application.php:1000 { …}
      C:\dev\web\project\vendor\symfony\symfony\src\Symfony\Bundle\FrameworkBundle\Console\Application.php:86 { …}
      C:\dev\web\project\vendor\symfony\symfony\src\Symfony\Component\Console\Application.php:255 { …}
      C:\dev\web\project\vendor\symfony\symfony\src\Symfony\Bundle\FrameworkBundle\Console\Application.php:74 { …}
      C:\dev\web\project\vendor\symfony\symfony\src\Symfony\Component\Console\Application.php:148 { …}
      C:\dev\web\project\vendor\contao\manager-bundle\bin\contao-console:48 { …}
    }
  },
  "command" => "debug:config NelmioCorsBundle -vvv",
  "message" => "Impossible to call set() on a frozen ParameterBag."
]
```

For me the dependency-container is required by MetaModels. I am happy to provide further information.

**Comment by richardhj** [comment]
For MetaModels, I started to remove legacy dependencies (including on cca/dependency-container) in MetaModels/core#1314.
For MultiColumnWizard, there is only one single reference on the translator, maybe we can remove the dependency on cca/translator @stefanheimes ?

**Comment by richardhj** [comment]
This line BTW should be trigger: contao-community-alliance\dependency-container\src\CcaDependencyInjectionBundle.php:40

**Comment by richardhj** [comment]
See #17

**Comment by baumannsven** [comment]
Fixed in 7447644975fc6b1646ffc746f06f9e4c4949af50
